distribution/Makefile

```diff
@@ -0,0 +1,21 @@
+sd-install:
+	cp *.service /etc/systemd/system/
+	cp *.socket /etc/systemd/system/
+	systemctl daemon-reload
+	
+sd-start:
+	systemctl enable osbuild-composer.socket
```

Comment:
Very minor nit: maybe we don't need "enable" here? Or just make it a separate target for convenience?